### DISCUSSION TITLE
blog/follow-is-review/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#360] [Announcements]
# blog/follow-is-review/

分享我几个月来使用 Follow.is 的体验，解析优缺点及实用功能，助你评估是否值得尝试。

https://jimmysong.io/blog/follow-is-review/

**Comment by abuxliu** [comment]
大佬 有没有邀请码

**Reply by rootsongjc** [reply]
我之前有几个邀请码，不过都已经送出去了。

**Comment by rootsongjc** [comment]
最近发现一个 bug：已阅的文章在选择只显示未读时，时不时仍会显示出来。而且，mac 中的 App 角标也会显示未读数字，但实际上早就没有未读文章了。

**Comment by rootsongjc** [comment]
这个工具已经改名叫 Folo 了，可以在 AppStore 下载。https://follow.is/
